src/app/app.component.ts

```diff
@@ -1,10 +1,29 @@
 import { Component } from '@angular/core';
+import { NavigationStart, Router } from '@angular/router';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'technical-assistant';
+  title = 'navbar';
+  showHead: boolean = false;
+
+  ngOnInit() {
+  }
+
+  constructor(private router: Router) {
+    router.events.forEach((event) => {
+      if (event instanceof NavigationStart) {
+        if (event['url'] == '/login') {
```

Comment:
Can we use constants for the router URL?

Reply:
We already changed this